dissect/cstruct/types/structure.py

```diff
@@ -401,6 +411,49 @@ def __repr__(self) -> str:
 
         return f"<{self.__class__.__name__} {' '.join(values)}>"
 
+    @property
+    def __sizes__(self) -> Mapping[str, int | None]:
+        return ChainMap(self.__class__.__static_sizes__, self.__dynamic_sizes__)
+
+    @property
+    def __values__(self) -> MutableMapping[str, Any]:
+        return StructureValuesProxy(self)
```

Comment:
```suggestion
    @property
    def __values__(self) -> MutableMapping[str, Any]:
        return StructureValuesProxy(self)

    @property
    def __sizes__(self) -> Mapping[str, int | None]:
        return ChainMap(self.__class__.__static_sizes__, self.__dynamic_sizes__)
```
Just for consistency sake.